.eslintrc.js

```diff
@@ -23,38 +17,35 @@ module.exports = {
     ecmaVersion: 11,
     sourceType: 'module',
   },
-  plugins: [
-    'react',
-    '@typescript-eslint',
-  ],
+  plugins: ['react', '@typescript-eslint'],
   rules: {
-    semi: ['error', 'never'],
+    semi: ['error', 'always'],
```

Comment:
I think it is common to always use `semi`. It may be just some legacy rule setting so I changed that